exaudfclient/base/javacontainer/script_options/parser_ctpg_script_importer.cc

```diff
@@ -43,52 +77,28 @@ void ScriptImporter::importScript(std::string & scriptCode,
                                   {
                                       return first.idx_in_source < second.idx_in_source;
                                   });
-        struct ReplacedScripts {
-            ReplacedScripts(ReplacedScripts&&) = default;
-            std::string script;
-            size_t origPos;
-            size_t origLen;
-        };
-        std::vector<ReplacedScripts> replacedScripts;
-        replacedScripts.reserve(optionIt->second.size());
+        std::vector<CollectedScript> collectedScript;
+        collectedScript.reserve(optionIt->second.size());
         //In order to continue compatibility with legacy implementation we must collect import scripts in forward direction
```

Comment:
this should probably go into collectImportScripts

Reply:
Done